packages/manager/modules/carbon-calculator/src/index.scss

```diff
@@ -4,10 +4,15 @@
 @import 'bootstrap4/scss/_utilities.scss';
 @import 'bootstrap4/scss/_grid.scss';
 
+.carbon-balance-spacings {
+  *:not(:last-child) {
+    margin-bottom: 0px;
+  }
+}
 .carbon-balance {
   ul {
+    margin-bottom: 5px;
```

Comment:
As we're using bootstrap, use its own classes --> https://getbootstrap.com/docs/4.0/utilities/spacing/